domain-models-runtime/src/main/resources/templates/db_scripts/optimistic_locking.ftl

```diff
@@ -14,7 +14,7 @@
           <#if table.withOptimisticLocking.name() == "FAIL">
             <#assign ol_notice_level = "EXCEPTION">
           <#else>
-            <#assign ol_notice_level = "WARNING">
+            <#assign ol_notice_level = "NOTICE">
           </#if>
           RAISE ${ol_notice_level} 'Cannot update record % because it has been changed: '
               'Stored ${ol_version} is %, ${ol_version} of request is %',
```

Comment:
There should be different messages for EXCEPTION and NOTICE, see comment above.

Can you also add TABLE and SCHEMA information in the USING clause for a complete log entry?